ansible/roles/postgresql/templates/postgresql.conf.j2

```diff
@@ -65,7 +65,7 @@ listen_addresses = '*'      # what IP address(es) to listen on;
                     # defaults to 'localhost', '*' = all
                     # (change requires restart)
 port = 5432             # (change requires restart)
-max_connections = 100           # (change requires restart)
+max_connections = 300           # (change requires restart)
```

Comment:
ah ok cool. i downloaded [pgbadger](http://dalibo.github.io/pgbadger/) and ran it on the last 7 days of our postgres logs. i can send you the html report if you're interested. it actually has some pretty cool stuff in it. i can't point to anything specific that would cause it, but you might find something useful. pgbouncer sounds like the right way to go